### PR TITLE
Descriptions

### PR DESCRIPTION
**Pull request by FordFriedel**
changes /help

### CODE REVIEW

File: src/slashcommands/Help.ts [hunk 1]
```diff
@@ -18,16 +19,22 @@ export class Help implements SlashCommand {
...
 		Permissions.FLAGS.MANAGE_MESSAGES,
 		Permissions.FLAGS.ADD_REACTIONS,
 	];
 	async run(bot: Bot, interaction: CommandInteraction): Promise<void> {
 		try {
+			type cmdList = {[index:string]: string};
+			let cmds = {} as cmdList;
+			bot.slashCommands.forEach((command)=>{
+				cmds[command.name] = command.description;
+				console.log(`${command.name}: ${command.description}`);
```
Comment (CPhantasm) [inline src/slashcommands/Help.ts]: I will never merge a pr that contains console.log

File: src/slashcommands/Help.ts [hunk 2]
```diff
@@ -18,16 +19,22 @@ export class Help implements SlashCommand {
...
-		Permissions.FLAGS.EMBED_LINKS,
+		Permissions.FLAGS.EMBED_LINKS, 
 		Permissions.FLAGS.MANAGE_MESSAGES,
 		Permissions.FLAGS.ADD_REACTIONS,
 	];
 	async run(bot: Bot, interaction: CommandInteraction): Promise<void> {
 		try {
+			type cmdList = {[index:string]: string};
+			let cmds = {} as cmdList;
+			bot.slashCommands.forEach((command)=>{
```
Comment (CPhantasm) [inline src/slashcommands/Help.ts:30]: I don't exactly like that you're looping through all the commands just to assign them to an object again, we can find the command by name in the array, but I understand that it makes accessing them easier in bulk

File: src/slashcommands/RemoveIntro.ts [hunk 3]
```diff
@@ -15,7 +15,7 @@ import { Option, Subcommand } from './Option';
 
 export class RemoveIntro implements SlashCommand {
 	public name = 'removeintro';
-	description: string = '[MANAGER] Delete someones intro theme!';
+	description: string = '[MANAGER] =Remove someones intro theme!';
```
Comment (CPhantasm) [inline src/slashcommands/RemoveIntro.ts:18]: =Remove?

**Review by CPhantasm — CHANGES_REQUESTED** [review]

File: src/slashcommands/Help.ts (hunk 1 above)
Comment (FordFriedel) [reply src/slashcommands/Help.ts]: xD

**Review by CPhantasm — APPROVED** [review]

**Pull request merged**